lib/check-self-update/display-update-message.js

```diff
@@ -0,0 +1,8 @@
+const message = ({name, currentVersion, latestVersion}) => [
```

Comment:
@rafaelcalpena This module has an implicit dependency on https://github.com/LabShare/lsc/blob/master/cli/init.js#L39 executing first. It would be better to display the actual message in https://github.com/LabShare/lsc/blob/master/lib/cli/start.js. Then you can use `app.log.info()`.